python/tskit/tables.py

```diff
@@ -374,6 +374,43 @@ def __getitem__(self, index):
 
         return ret
 
+    def __setitem__(self, index, new_row):
+        """
+        Replaces a row of this table at the specified index with information from a
+        row-like object. Metadata, will be validated and encoded according to the table's
+        :attr:`metadata_schema<tskit.IndividualTable.metadata_schema>`.
+
+        :param index: the zero-index of the row to change
```

Comment:
Maybe delete the zero here? All indexes are zero-based, might just be confusing.

Reply:
This was cut and pasted from `__getitem__` so I've removed it there also.